examples/distributed_example.py

```diff
@@ -4,6 +4,16 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+"""
```

Comment:
did you accidentally check in this file?

Reply:
yep. sorry, my bad. Let me clean this file.